### PR TITLE
add more style[include] doc

### LINKED ISSUE #3601: Make a style[include] doc better
There should be a code example of `<style include="module-name">` in the `custom-style` doc.

We should also note explicitly that multiple `dom-module`s can be `include`d (i.e. `include="module1 module2"`), so people don't have to go searching through the source code or [unit tests](https://github.com/Polymer/polymer/blob/edb59eb6283074ce54f374b30b884c9f9fcf9f5f/test/unit/custom-style-import.html#L23) to figure out if that's possible ;).

### Versions
- Polymer: v1.1.0+

/cc @sorvell

### PR DESCRIPTION
**Pull request by danbeam**
### Reference Issue
Fixes #3601

### CODE REVIEW

File: src/lib/custom-style.html [hunk 1]
```diff
@@ -54,7 +54,27 @@
     }
 
   </style>
-
+  
+  
+  <!-- This could easily live in another .html file and be imported instead. -->
```
Comment (sorvell) [inline src/lib/custom-style.html]: This actually ends the HTML comment so the example below is not commented.

Comment (danbeam) [inline src/lib/custom-style.html]: Just removed that comment for now.

**Comment by sorvell** [comment]
LGTM, thx!

**Pull request merged**